### PR TITLE
OHARA-1555 Change README.md to fit the latest Ohara start logic

### PR DESCRIPTION
**Pull request by saivirtue**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -79,7 +86,7 @@ docker run --rm -p 5050:5050 oharastream/manager:0.3-SNAPSHOT --port 5050 --conf
 ```
 gradle clean test
 ```
-NOTED: Some tests in ohara-it require "specified" env. Otherwise, they will be skipped.
+**NOTED:** Some tests in ohara-it require "specified" env. Otherwise, they will be skipped.
```
Comment (konekoya) [inline README.md]: Hmm, I think it should specific? And what does "env" mean here? Environment variables? If that's the case, then env variables or environment variables would better :)

File: README.md [hunk 2]
```diff
@@ -47,25 +47,32 @@ The backend image is not included in production release. Hence, there is no guar
 #### MySQL
 
 ```
-docker run --rm -p 20000:20000 oharastream/backend:0.3-SNAPSHOT con.island.ohara.integration.Database --port 20000
+docker run --rm -p ${port}:${port} oharastream/backend:0.3-SNAPSHOT com.island.ohara.integration.Database --port ${port} --user ${USERNAME} --password ${PASSWORD}
 ```
+* port: bound by MySQL
+* user: username for connecting to MySQL
+* password: password for connection to MySQL
```
Comment (konekoya) [inline README.md]: *connecting* or simply *username for MySQL and password for MySQL*

File: README.md [hunk 3]
```diff
@@ -47,25 +47,32 @@ The backend image is not included in production release. Hence, there is no guar
...
 ```
 * port: bound by Configurator (default is random)
-* brokers: broker information (ex. host0:port0,host1:port1)
-* workers: worker information (ex. host0:port0,host1:port1)
+* add-host: adding a host mapping to /etc/hosts in configurator (nodeHostName:nodeHostIP)
+* node: running a configurator with "pre-created" broker and zookeeper clusters (for testing purpose)
 
-If either brokers or workers are not defined, the configurator will be running with no-cluster mode. It means all data are 
-stored in memory, and connector-related commands are executed by nothing.
+If `node` property is not defined, the configurator will be running with no-cluster mode. You will need to add clusters
```
Comment (konekoya) [inline README.md]: *is not specified*

File: README.md [hunk 4]
```diff
@@ -47,25 +47,32 @@ The backend image is not included in production release. Hence, there is no guar
...
+by APIs.
+
+**NOTED:** you should pull the broker and zookeeper image in each node for pre-created clusters if you added the `node` property
```
Comment (konekoya) [inline README.md]: *images*

File: README.md (hunk 1 above)
Comment (saivirtue) [reply README.md]: use env variables instead. thanks!

**Comment by konekoya** [comment]
I'm okay with these changes but I think maybe @chia7712 or @jackyoh should have a quick as well?

**Comment by chia7712** [comment]
> should have a quick 

英文小教室時間：請問這句是什麼意思？

**Comment by konekoya** [comment]
look XD

**Comment by saivirtue** [comment]
another small question...
should we add "required images of nodes" in Installation section?

**Comment by chia7712** [comment]
> should we add "required images of nodes" in Installation section?

+1

**Comment by saivirtue** [comment]
@vitojeng @chia7712 @konekoya 
I have updated the README.md for including some installation, please help review.

File: README.md [hunk 5]
```diff
@@ -23,6 +23,17 @@ gradle licenseApply
 
 ### Installing
 
+**Running Ohara In Docker**
+- - - 
+* Ensure your nodes (actual machine or VM) are installed Docker 18.09+
```
Comment (chia7712) [inline README.md]: please don't forget connector...worker-connector image is required also

Comment (chia7712) [reply README.md]: ditto for configurator image and manager image

File: README.md [hunk 6]
```diff
@@ -89,7 +108,7 @@ gradle clean build -PskipManager
 
 ### build uber jar
 ```
-gradle clean uberJar
+gradle clean uberJar -PskipManager
```
Comment (chia7712) [inline README.md]: good

**Pull request merged**